DOCS: Improve custom admonitions documentation (#1777)
executablebooks/jupyter-book: docs/advanced/html.md

:::

Currently, using the `{admonition}` directive with a title creates a CSS class based on the title of the admonition.
For example, an admonition title of `Here's my title` will result in a class name of `admonition-here-s-my-title`.

On the other hand, by using the `:class:` keyword argument, it will create a class with the keyword previously chosen.
For example, a custom class defined as `my-custom-class` will result in a class named as `my-custom-class`.

> You can leverage this pattern to quickly create custom admonitions.
> For example, create a `myadmonitions.css` under `mybook/_static` with the following CSS:

Reviewer: ```suggestion
You can leverage either of these patterns to quickly create custom admonitions.
There is an example of each below.
In each case, begin by creating a `myadmonitions.css` file under `mybook/_static` and add CSS rules to it.
```

> #### Using the `{admonition}` directive with a title

Reviewer: ```suggestion
**Using the `{admonition}` directive with a title**
```

```css

> div.admonition-extra-credit {

Reviewer: why add the `div`?  I think we may not want to do this because different versions of `sphinx` make this a `div` vs. an `aside`

Author: That's because the .css files from sphinx would overlap the custom .css, since they have the .div in front and thus being more specific.

@brownsarahm opened an issue related to it (#1558) and, after my PR, noticed that it also solved her issue.

    border-left-color: rgba(0, 246, 16, 1);
}
div.admonition-extra-credit .admonition-title {
    background-color: rgba(0, 246, 16, .1);
}
div.admonition-extra-credit .admonition-title:before {
    color: rgba(0, 246, 16, 1);
    content: '\f19d';
}
```

Then, in your book, define an admonition like so:

````md
```{admonition} Extra credit
An "extra credit" exercise is presented here.
```
````

> #### Using the `:class:` keyword argument

Reviewer: ```suggestion
**Using the `:class:` keyword argument**
```

```css
div.extra-credit {
    border-left-color: rgba(var(--pst-color-success), 1);
}
div.extra-credit .admonition-title {
    background-color: rgba(var(--pst-color-success), .1)
}
div.extra-credit .admonition-title:before {
    color: rgba(var(--pst-color-success), 1);
    content: '\f19d';
}
```

Then, in your book, define an admonition like so:

````md
```{admonition} An extra exercise
:class: extra-credit
An "extra credit" exercise is presented here.
```
````

> On both cases the admonitions should be styled according to your CSS rules when you build your book.

Reviewer: ```suggestion
In both cases the admonitions should be styled according to your CSS rules when you build your book.

```
